src/js/common/components/Modal.jsx

```diff
@@ -1,10 +1,38 @@
 import React from 'react';
 import classNames from 'classnames';
 
+function focusListener() {
+  const listener = event => {
+    const modal = document.querySelector('.va-modal');
+    if (!modal.contains(event.target)) {
+      event.stopPropagation();
+      modal.focus();
```

Comment:
Actually, never mind. When the other modals are hidden, they're just a plain `div` without the `va-modal` class. Should be good.